locales/en-US/server.ftl

```diff
@@ -0,0 +1,214 @@
+// Global phrases shared across pages, prefixed with 'g'
+gMyShots = My Shots
+gHomeLink = Home
+gNoShots
+  .alt = No shots found
+gScreenshotsDescription = Screenshots made simple. Take, save, and share screenshots without leaving Firefox.
+
+// Creating page
+
+// Note: {$title} is a placeholder for the title of the web page
+// captured in the screenshot. The default, for pages without titles, is
+// creatingPageTitleDefault.
+creatingPageTitle = Creating {$title}
+creatingPageTitleDefault = page
+
+// Home page
+
+homePageDescription
+  .content = Intuitive screenshots baked right into the browser. Capture, save and share screenshots as you browse the Web using Firefox.
+homePageButtonMyShots = Go To My Shots
+homePageTeaser = Coming Soon…
+homePageDownloadFirefoxTitle = Firefox
+homePageDownloadFirefoxSubTitle = Free Download
+homePageGetStarted = Get Started
+// Note: do not translate 'Firefox Screenshots' when translating this string
+homePageHowScreenshotsWorks = How Firefox Screenshots Works
+homePageGetStarted = Get Started
```

Comment:
Probably homePageGetStartedTitle to differentiate from the other one, which is a link?